pkgs/servers/computing/slurm/default.nix

```diff
@@ -10,18 +10,26 @@ stdenv.mkDerivation rec {
     sha256 = "05si1cn7zivggan25brsqfdw0ilvrlnhj96pwv16dh6vfkggzjr1";
   };
 
-  buildInputs = [ pkgconfig curl python munge perl pam openssl mysql.lib ncurses gtk ];
+  buildInputs = [ pkgconfig curl python munge perl pam openssl mysql.lib 
+		  ncurses gtk lua hwloc numactl ] ++ extraDeps;
 
   configureFlags =
     [ "--with-munge=${munge}"
       "--with-ssl=${openssl}"
+      "--sysconfdir=/etc/slurm"
```

Comment:
It should but it is not :)
In fact, all the main distribution including the official [slurm spec file ](https://github.com/SchedMD/slurm/blob/master/slurm.spec), RedHat, Fedora and Debian override the %sysconfdir to /etc/slurm due to the quite large numbers of conf files of slurm.